### PR TITLE
add hidefix

### PR DESCRIPTION
**Pull request by gauteh**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by gauteh** [comment]
Hi @conda-forge/staged-recipes, this is ready for review!

**Comment by gauteh** [comment]
Hi https://github.com/orgs/conda-forge/teams/staged-recipes, please take a look.

**Comment by ocefpaf** [comment]
xref.: https://github.com/conda-forge/staged-recipes/pull/21742

@xhochy it seems you had some blocking comments in the original PR. Does this one addresses your concerns?

@gauteh closing and re-submitting puts an extra strain in our volunteer reviewers. Please try to keep things in the same thread/PR the next time.

PS: I'm not sure those `missing_dso_whitelist` DLLs are OK. I kind of recall some suggestions in the original PR. Did you try them?

File: recipes/hidefix/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hidefix-{{ version }}.tar.gz
+  sha256: a9162d5f881e4b9890d8852f543b85e77ab565ae6b52bfe7420e64002fa3a5ed
+
+build:
+  number: 0
+  # I think all these files are standard windows files, and not really provided by a package. They are probably linked in
+  # by some package providing random numbers. Since they are standard windows files, they should be safe to ignore?
+  missing_dso_whitelist:
```
Comment (xhochy) [inline recipes/hidefix/meta.yaml:16]: Adding the c compiler on Windows should fix this.

File: recipes/hidefix/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    - maturin                             # [build_platform != target_platform]
+    - {{ compiler('c') }}                 # [linux]
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - cargo-bundle-licenses
+  host:
+    - python
+    - pip
+    - maturin >=0.15,<0.16
+    - pkg-config
```
Comment (xhochy) [inline recipes/hidefix/meta.yaml:41]: Should only be in build 
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/hidefix/meta.yaml (hunk 1 above)
Comment (gauteh) [reply recipes/hidefix/meta.yaml]: Thanks. Trying.

Comment (gauteh) [reply recipes/hidefix/meta.yaml]: Seems to have done the trick.

**Comment by gauteh** [comment]
Hi @conda-forge/staged-recipes, this is ready for review

File: recipes/hidefix/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - cmake
+    - make
+    - pkg-config
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - maturin                             # [build_platform != target_platform]
```
Comment (xhochy) [inline recipes/hidefix/meta.yaml:25]: You sound be setting the same constraints as in host here

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/hidefix/meta.yaml (hunk 3 above)
Comment (gauteh) [reply recipes/hidefix/meta.yaml]: on the version? added.

**Review by xhochy — APPROVED** [review]

**Comment by gauteh** [comment]
Thank you for the review and merging!

**Pull request merged**